test-js/integration.js

```diff
@@ -192,6 +192,13 @@ describe("integration tests", function() {
     assert.exists(path.join("output", "Main", "index.js"));
   }));
 
+  notWindowsIt("pulp build with psc-package", run(function*(sh, pulp, assert, temp) {
```

Comment:
I actually tried AppVeyor for psc-package out of curiosity, with great inspiration from your PS appveyor.yml ^_^ I'm gonna do a PR for it and we'll see if psc-package can turn it on soon?